### PR TITLE
Cumulative Update for Run2_UL Changes

### PR DESCRIPTION
**Pull request by JHiltbrand**
This PR will contain changes as a reaction to changes in TreeMaker for UL and just changes for UL in general

--update DeepCSV WPs for the different years
--update the latest lumi numbers for 2016 and 2017
--removed EE noise analyzer, EE noise taken care of in UL [1]
--removed old W control region and Njets analyzer specific to early days of SUS-19-004
--update all analyzers to use new `utility::LorentzVector`
--update correct types when getting variables from the framework

[1] https://twiki.cern.ch/twiki/bin/view/CMS/JetMET#Quick_links_to_current_recommend

### CODE REVIEW

File: Analyzer/test/Makefile.in [hunk 1]
```diff
@@ -40,7 +40,7 @@ LIBSTOPTAGGER += -L$(TTTDIR) -lTopTagger
 INCLUDESDIRS  += $(shell root-config --cflags)
 
 # ROOT libraries
-LIBS          += $(shell root-config --glibs)
+LIBS          += $(shell root-config --glibs) -lGenVector
```
Comment (cmadrid1) [inline Analyzer/test/Makefile.in:43]: did you need to add `-lGenVector`? I thought it was needed but I think it is depreciated from like ROOT 5.

File: Analyzer/include/Histo.h [hunk 2]
```diff
@@ -6,6 +6,8 @@
 #include <iostream>
 #include <string>
 
+#include "TH3D.h"
```
Comment (cmadrid1) [inline Analyzer/include/Histo.h:9]: Was this needed? Just curious doesn't really matter either way.

File: Analyzer/include/Histo.h [hunk 3]
```diff
@@ -118,7 +120,7 @@ class Histo1D : public Histo_FirstChild<TH1>
             else if(type.find("char")           != std::string::npos) vecFill<char>        ( weight, tr );
             else if(type.find("short")          != std::string::npos) vecFill<short>       ( weight, tr );
             else if(type.find("long")           != std::string::npos) vecFill<long>        ( weight, tr );        
-            else if(type.find("TLorentzVector") != std::string::npos) vecFilltlv           ( weight, tr );        
+            else if(type.find("utility::LorentzVector") != std::string::npos) vecFilltlv           ( weight, tr );
```
Comment (cmadrid1) [inline Analyzer/include/Histo.h:123]: Does this work? I wasn't sure if the fundamental name of the type changes with the typedef.

Comment (cmadrid1) [reply Analyzer/include/Histo.h:123]: Also, we shouldn't remove the TLorentzVector option from this file.

File: Analyzer/test/Makefile.in (hunk 1 above)
Comment (JHiltbrand) [reply Analyzer/test/Makefile.in:43]: yeah, early on in these changes, I could not get things to compile (undefined reference when I wanted to include for the ROOT::Math::Boost method, I believe)

File: Analyzer/include/Histo.h (hunk 3 above)
Comment (JHiltbrand) [reply Analyzer/include/Histo.h:123]: I will put back

File: Analyzer/include/Histo.h (hunk 2 above)
Comment (JHiltbrand) [reply Analyzer/include/Histo.h:9]: When cleaning up some includes (due to switch away from `TLorentzVector`) I needed to explicitly include `TH3D.h` as the methods were no longer getting included from wherever they were coming from before.

File: Analyzer/include/Histo.h (hunk 3 above)
Comment (JHiltbrand) [reply Analyzer/include/Histo.h]: Done in ad24937

**Pull request merged**